src/data/waypoint.json

```diff
@@ -1,29 +1,30 @@
 {
   "subnavItems": [
     {
-      "id": "reference-docs",
```

Comment:
Updated to match latest work in Figma.

Reply:
Could you link to the Figma designs you're referencing? I didn't know there were changes and I just want to make sure I'm up to date. 😊

Reply:
For sure - I believe it's the same `Dev Portal` file ([link](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=536%3A39227)) and page (`Page Templates`). I saw some work happening in there yesterday, and when I ran the preview links from this PR by design, there was some QA related to recent changes.

Reply:
I'm seeing three different things there so I'm not sure which is the right one 😆

- ["Reference Docs"](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=1377%3A50800)
- ["Docs"](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=3241%3A65842)
- ["Reference"](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=3274%3A66112)